### PR TITLE
Add runbook on binderhub-ui hub

### PR DESCRIPTION
**Pull request by GeorgianaElena**
Fixes https://github.com/2i2c-org/infrastructure/issues/4169

### CODE REVIEW

File: docs/howto/features/binderhub-ui.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        #        search after the wrong secret name (i.e. the default name)
+        #        set by binderhub in KubernetesBuildExecutor.push_secret
+        01-binderhub-service-set-push-secret: |
+          import os
+          c.KubernetesBuildExecutor.push_secret = os.environ["PUSH_SECRET_NAME"]
+      extraEnv:
+        - name: JUPYTERHUB_API_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: '{{ include "jupyterhub.hub.fullname" . }}'
```
Comment (yuvipanda) [inline docs/howto/features/binderhub-ui.md:144]: Let’s just say “hub” here insteaf of templating. It will never be a different value for us.

Comment (yuvipanda) [reply docs/howto/features/binderhub-ui.md:144]: This template name i also consider an internal implementation detail i don’t want to dive into and depend on

**Comment by yuvipanda** [comment]
I'm writing out new code review guidelines that should help cover this (and we can discuss that during our 1:1 tomorrow). I haven't read through this, but I suggest the following course of action:

1. Merge it
2. Follow it to do https://github.com/2i2c-org/infrastructure/issues/4134
3. Make changes as you find issues, as well as make additions for the two ways in which that is different from binderhub-ui-demo
4. Make a PR with those changes

**Comment by GeorgianaElena** [comment]
Good plan @yuvipanda! Thank you. Going for a merge then 🚀

**Pull request merged**